test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerUnitTestabilityTests.cs

```diff
@@ -34,12 +34,16 @@ public void ControllerView_InvokedInUnitTests(object model, string viewName)
             var viewResult = Assert.IsType<ViewResult>(result);
             Assert.Equal(viewName, viewResult.ViewName);
             Assert.NotNull(viewResult.ViewData);
+            Assert.Same(model, viewResult.Model);
             Assert.Same(model, viewResult.ViewData.Model);
             Assert.Same(controller.ViewData, viewResult.ViewData);
             Assert.Same(controller.TempData, viewResult.TempData);
 
             if (model != null)
             {
+                Assert.IsType(model.GetType(), viewResult.Model);
+                Assert.NotNull(viewResult.Model);
+
                 Assert.IsType(model.GetType(), viewResult.ViewData.Model);
                 Assert.NotNull(viewResult.ViewData.Model);
             }
```

Comment:
I'm not really sure what the purpose of this block is.. this is already covered by L38 in the existing code, and L37-38 - suggest deleting this